.github/workflows/acceptance-test.yml

```
@@ -52,7 +52,7 @@ jobs:
     with:
       ref: ${{ github.ref }}
       version: ${{ inputs.version || needs.version.outputs.version }}
-      short_test: ${{ inputs.short_test || false }}
+      short_test: ${{ (github.event_name == 'pull_request' || inputs.short_test) || false }}
```

I think this can only be reached via the workflow call or the pull request triggers. The workflow call should always set the `inputs.short_test`, so I think we can just change the fallback:

```suggestion
      short_test: ${{ (inputs.short_test) || true }}
```